tests/acceptance/homepage/en-us-test.js

```diff
@@ -23,4 +23,22 @@ module('Acceptance | Homepage | en-US', function(hooks) {
     assert.dom(document.querySelector('html'))
       .hasAttribute('lang', 'en-us', 'We set the correct lang attribute.');
   });
-});
\ No newline at end of file
+
+  test('We can change the display language', async function(assert) {
+    await visit('/');
+    assert.dom(document.querySelector('#generating-files'))
+      .hasText('§ Generating Files');
+
+    await fillIn(document.querySelector('select[data-test-field="Locale"]'), 'pt-BR');
+    assert.dom(document.querySelector('#generating-files'))
+      .hasText('§ Geração de arquivos');
+
+    await fillIn(document.querySelector('select[data-test-field="Locale"]'), '');
+    assert.dom(document.querySelector('#generating-files'))
+      .hasText('§ Geração de arquivos');
+
+    await fillIn(document.querySelector('select[data-test-field="Locale"]'), 'fr-FR');
+    assert.dom(document.querySelector('#generating-files'))
+      .hasText('§ Génération de fichiers');
+  });
```

Comment:
Since we made multiple assertions in the test, let's add descriptions to make the test easier to understand.

```suggestion
  test('We can change the site language', async function(assert) {
    await visit('/');
    assert.dom(find('#generating-files'))
      .hasText('§ Generating Files', 'We see the site in English.');

    await fillIn('[data-test-field="Locale"], 'pt-BR');
    assert.dom(find('#generating-files'))
      .hasText('§ Geração de arquivos', 'We see the site in Portugese (Brazil).');

    await fillIn('[data-test-field="Locale"]', '');
    assert.dom(find('#generating-files'))
      .hasText('§ Geração de arquivos', 'We still see the site in Portugese (Brazil).');

    await fillIn('[data-test-field="Locale"]', 'fr-FR');
    assert.dom(find('#generating-files'))
      .hasText('§ Génération de fichiers', 'We see the site in French.');
  });
```